.github/ISSUE_TEMPLATE/form404.yml

```diff
@@ -0,0 +1,32 @@
+name: Broken Link Report
+description: File a broken link report
+title: "Broken Link:"
+labels: ["bug"]
+projects: ["ros-infrastructure/rosindex"]
+body:
+
+  - type: input
+    id: 404page
+    attributes:
+      label: 404 Page URL
+      description: If this isn't autodetected it means you have to fill it in yourself
+      placeholder: FAILED TO AUTOMATICALLY DETECT 404 PAGE URL PLEASE FILL
+    validations:
+      required: true
+
+  - type: input
+    id: referer
+    attributes:
+      label: Referrer URL
+      description: If this isn't autodetected it means you have to fill it in yourself
+      placeholder: FAILED TO AUTOMATICALLY DETECT REFERRER URL PLEASE FILL
+    validations:
+      required: true
+
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      placeholder: If any additional information is needed, please provide it here.
```

Comment:
```suggestion
      placeholder: Please describe where the link is that you followed if it might not be obvious on the referring page. If any additional information is needed, please provide it here too.
```